spec/requests/graphql/users_query_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "users query" do
```

Comment:
Can you add an example of mutation? 👀

Reply:
issue was created for this https://github.com/espoo-dev/rails_boilerplate/issues/146